src/main/java/org/bookkepervisualmanager/api/ApplicationConfig.java

```diff
@@ -36,6 +36,7 @@
     private void addRestResourceClasses(Set<Class<?>> resources) {
         resources.add(org.bookkepervisualmanager.api.resources.BookiesResource.class);
         resources.add(org.bookkepervisualmanager.api.resources.LedgersResource.class);
+        resources.add(org.glassfish.jersey.server.wadl.internal.WadlResource.class);
```

Comment:
?

Reply:
I guess this is your IDE that is very "creative"